features/networking/ib-sriov.feature

```diff
@@ -0,0 +1,158 @@
+Feature: Sriov IB related scenarios
+
+  # @author zzhao@redhat.com
+  @destructive
+  @admin
+  Scenario Outline: IPoIB and IB mode for CX4/CX5/CX6 card
+    Given the sriov operator is running well
+    Given I obtain test data file "networking/sriov/ib/<cardname>/<cardname>-ib.yaml"
```

Comment:
Do we need a real car name or just <cardname> here?

Reply:
yes, see line 63-65

Reply:
Get it, thanks!